src/aggregator/aggregator/handler/config.go

```diff
@@ -237,7 +314,7 @@ func (c *dynamicBackendConfiguration) NewSharderRouter(
 		r = trafficcontrol.NewRouter(tc, r, scope)
 	}
 	return SharderRouter{
-		SharderID: sharding.NewSharderID(c.HashType, c.TotalShards),
+		SharderID: sharding.NewSharderID(*c.HashType, *c.TotalShards),
```

Comment:
Is it ok to dereference these fields without checking if they `nil` first? In `newHandler` you first check that `c.ProtobufEnabled` is not `nil` before dereferencing it.

Reply:
done